autonomi/src/client/data_types/pointer.rs

```diff
@@ -52,11 +52,32 @@ pub enum PointerError {
 impl Client {
     /// Get a pointer from the network
     pub async fn pointer_get(&self, address: PointerAddress) -> Result<Pointer, PointerError> {
+        let result = self.pointer_get_inner(address).await;
+        // Reporting
+        if let Some(channel) = self.client_event_sender.as_ref() {
+            let xor_name = address.xorname();
+            let event = if result.is_ok() {
+                ClientEvent::DownloadSucceeded(*xor_name)
+            } else {
+                ClientEvent::DownloadFailed(*xor_name)
+            };
+
+            if let Err(err) = channel.send(event).await {
+                error!("Failed to send client event: {err:?}");
+            }
+        }
+        result
+    }
+
+    async fn pointer_get_inner(&self, address: PointerAddress) -> Result<Pointer, PointerError> {
         let key = NetworkAddress::from_pointer_address(address).to_record_key();
         debug!("Fetching pointer from network at: {key:?}");
         let get_cfg = GetRecordCfg {
-            get_quorum: Quorum::Majority,
-            retry_strategy: Some(RetryStrategy::Balanced),
+            get_quorum: self.operation_config.pointer_operation_config.read_quorum,
+            retry_strategy: self
+                .operation_config
+                .pointer_operation_config
+                .read_retry_strategy,
             target_record: None,
             expected_holders: Default::default(),
         };
```

Comment:
I don't see how this is improving our code. How do we know what quorum we are using now? Why isn't it a default by type?

Reply:
The configs are set inside a single file for all the data types `autonomi/src/client/config.rs`. 

I think both of these patterns have their own tradeoffs.. :/ For instance, with the old method we got to see the configs without indirections when we are editing in that area. But this also meant that if people are not looking into that particular area of code, sometimes incorrect values can creep in. I think prior to a couple rebases, I found places with 0 retries for verification.

Reply:
The other alternative is to set `Option` for the inner configs and override the default values with those `Some<config>` values. I tried it earlier, and I like the ergonomics of `config.rs`..